### PR TITLE
Proposing myself as LT for RHEL and Core

### PR DESCRIPTION
**Pull request by jonlives**
I'm proposing myself as LT for RHEL and Core - I've split the proposals into two commits so they can be handled separately if necessary.

### CODE REVIEW

**Comment by nathenharvey** [comment]
@sethvargo @btm @martinisoft can you please weigh in here, too?

File: MAINTAINERS.md [hunk 1]
```diff
@@ -22,6 +22,8 @@ another component.
 
 ### Lieutenant
 
+* [Jon Cowie](http://github.com/jonlives)
```
Comment (martinisoft) [inline MAINTAINERS.md]: If you're proposing LT, doesn't this also make you a maintainer by definition of the RFC?

**Comment by martinisoft** [comment]
Commented. Not sure you need to be both LT _and_ Maintainer as I thought LT's also bear the responsibilities of a maintainer?

**Comment by sethvargo** [comment]
:+1: modulo what Mr. Kalin said :smile:

**Comment by thommay** [comment]
@martinisoft @sethvargo I can see the lifetime of maintainer and LT being different, though - time commitments and so on might mean you want to step back from the LT role but remain a maintainer, so it might actually be clearer to have two separate lists.

**Comment by btm** [comment]
A lieutenant has "all the responsibilities of a Maintainer"

https://github.com/chef/chef-rfc/blob/master/rfc030-maintenance-policy.md#lieutenant

**Comment by btm** [comment]
What's not clear to me is if you can have more than one lieutenant for a component. I don't see any difference between maintainer, lieutenant, and project lead in that regard in rfc030. It would seem to make sense that you could have many maintainers, and only one lead, but that lieutenants could go either way. We should be clear about that.

**Comment by lamont-granquist** [comment]
Also :+1: but @adamhjk is the decider, and you and thom have competing LTs

**Comment by adamhjk** [comment]
You cannot have multiple LTs, as they involve veto power.

@jonlives should be the LT for RHEL. @thommay should be the LT for core.

**Comment by jonlives** [comment]
@adamhjk thanks for clarifying - my only concern with that call is that the entire appeal chain for the core parts of chef would consist of Chef Inc employees. This *is* kind of a moot point as of course the decider has the ultimate call, but it doesn't make things feel as impartial as might perhaps be nice. BTW - I don't limit an impartial LT to being myself necessarily, this just happens to be the issue I bring this up on :p

**Comment by tyler-ball** [comment]
:+1:

File: MAINTAINERS.md (hunk 1 above)
Comment (btm) [inline MAINTAINERS.md]: yeah, should remove yourself from maintainers when switching to LT.

**Comment by adamhjk** [comment]
I think two things mitigate:

1) core has the responsibility for areas that aren't covered by another lt. It isn't a thing that floats up - so RHEL decisions aren't overseen by core LT, but by the RHEL lt and the decider. 

2) we can change it with a pr. If we have a strong feeling that core not being a chef software employee, let's talk about it. We placed the community team directly reporting to me, and we are committed to using the public process internally.

**Comment by danielsdeleo** [comment]
:+1:

**Comment by jonlives** [comment]
@adamhjk agreed - Once the tests have passed on the commit removing myself as RHEL maintainer per @btm's suggestion I'll merge this since there is much thumbsuppery.

**Pull request merged**